frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -354,6 +354,11 @@ const LabworkStep = ({ protocol, step, stepSamples }: LabworkStepPageProps) => {
 					/>
 				</>
 			}
+			<a href={`/management/workflow-assignment?${SampleColumnID.QUEUED_STEPS}=${step.name}`}>
```

Comment:
We are using <Link> nearly everywhere ... this would be good to stick to the same.